docs/source/configuration.rst

```diff
@@ -88,6 +109,79 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the :code:`.sqlfluff` config files.
+
+Common rule configurations can be set in the :code:`[sqlfluff:rules]` section.
+
+For example:
+
+.. code-block:: cfg
+
+   [sqlfluff:rules]
+   tab_space_size = 4
+   max_line_length = 80
+   indent_unit = space
+   comma_style = trailing
+   allow_scalar = True
+   single_table_references = consistent
+   unquoted_identifiers_policy = all
+
+Rule specific configurations are set in rule specific subsections.
+
+For example, enforce that keywords are upper case by configuring the rule
+:class:`L010 <sqlfluff.core.rules.Rule_L010>`:
+
+.. code-block:: cfg
+
+    [sqlfluff:rules:L010]  # Keywords
+    capitalisation_policy = upper
+
+All possible options for rule sections are documented in :ref:`ruleref`.
+
+For an overview of the most common rule configurations that you may want to
+tweak, see `Default Configuration`_ (and use :ref:`ruleref` to find the
+available alternatives).
+
+Enabling and Disabling Rules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To disable individual rules, set :code:`exclude_rules` in the top level section
+of sqlfluff configuration. The value is a comma separated list of rule ids.
+
+For example, to disable the rules :class:`L010 <sqlfluff.core.rules.Rule_L022>`
+and :class:`L010 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    exclude_rules = L022, L027
+
+To enable individual rules, configure :code:`rules`, respectively.
+
+For example, to enable :class:`L010 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    rules = L027
+
+^ **TODO** does this make sense only if overriding an exclusion from a higher level
+configuration? Or what does this conf mean? How is it interpreted if
+:code:`exclude_rules = None` but :code:`exclude_rules != None`?
+
+Additionally, some rules have a special :code:`force_enable` configuration
+option, which allows to enable the given rule even for dialects where it is
+disabled by default. The rules that support this can be found in the
+:ref:`ruleref`.
+
+The default values can be seen in `Default Configuration`_.
+
+See also: :ref:`inline_ignoring_errors` and `Ignoring Errors & Files`_.
+(**TODO** aren't those two pretty much overlapping?)
```

Comment:
This is strange. Is it ok to just link to both at this point?

Reply:
Oh I don't think we should duplicate this content. One should just be a link to the other.

Not sure where is best lives? Under configuration or under rules? But since it's under rules already let's just add the link here and remove all the text.

Reply:
Oh it's already duplicated in both! Never noticed that before!

OK let's leave as is. I think just the one link to the one on this page is fine. Might be worth doing a review to ensure they are both saying the same thing and sync up if not.